### ISSUE TITLE
Make work with Sidekiq 5.0

### ISSUE THREAD

**Issue by asanghi** [mhfs/sidekiq-failures#111]
I think the names of some middleware on Sidekiq 5.0 have changed. 

```
There was an error while trying to load the gem 'sidekiq-failures'.
Gem Load Error is: uninitialized constant Sidekiq::Middleware::Server
Backtrace for gem load error is:
/Users/xxx/.rvm/gems/ruby-2.3.3/gems/sidekiq-failures-0.4.5/lib/sidekiq/failures.rb:70:in `block (2 levels) in <top (required)>'
```

**Comment by thomasnaude** [comment]
+ 1

**Comment by sesson** [comment]
+1

**Comment by giamma13** [comment]
any news ?

**Comment by krasnoukhov** [comment]
I don't use this gem anymore, so I don't feel like I'll be of any help regarding maintaining. @mhfs should we do another round of asking for maintainers?

**Comment by mhfs** [comment]
Probably. I'm currently not working with ruby and won't have the time to update this gem any time soon. If someone wants to step in, I'd be glad to provide access.

**Comment by gaurish** [comment]
I like to maintain this for my company @punchh should we transfer over the repo or fork it? What is the process to contribute & maintain this?

**Comment by mhfs** [comment]
@gaurish awesome. It's great to have help. If it doesn't change your interest, I'd suggest we start by giving you commit access to this repo so that you can work on the pending issues and updates.

That would be followed by push access to rubygems.org at some point.

And finally, if the above goes well, we can talk about repo transfer. I just don't feel comfortable starting with ownership transfer.

Let me know what you think.

Thanks

**Comment by phlegx** [comment]
There is a pull request [#112](https://github.com/mhfs/sidekiq-failures/pull/112) from another user...

**Comment by mhfs** [comment]
This is fixed on master after PR #113 got merged. I'd be awesome if we could get a few people testing their projects against master before I cut a new gem release. Thanks

**Comment by fidelisrafael** [comment]
See #114

**Comment by mhfs** [comment]
#114 has been merged. Thanks @petergoldstein. We good now?

**Comment by petergoldstein** [comment]
@mhfs I believe so.

**Comment by ziaulrehmandevsinc** [comment]
Waiting for the new release (Y)

**Comment by Soliah** [comment]
@mhfs I've just tested master and #114 works.

**Comment by petergoldstein** [comment]
@mhfs Are there any additional blockers to cutting a new gem version and pushing it to Rubygems?

**Comment by fidelisrafael** [comment]
I'm waiting for a new gem version too \o/

**Comment by dennislysenko** [comment]
Master seems to work 👍

**Comment by petergoldstein** [comment]
@mhfs Just checking in again - any blockers to cutting a new gem?  We're using `master` in production, but we'd prefer to use a released gem.

**Comment by mhfs** [comment]
Hi all,

Sorry for the delay, I was on vacation. I just released 1.0.0 to rubygems.org. Enjoy.

Also, I'm on the market for new full time or contract opportunities. Please let me know if any of your companies are hiring.

Best,
Marcelo
